### PR TITLE
Update fix spawner status keyword for KFNBC

### PR DESCRIPTION
**Pull request by lugi0**
Signed-off-by: Luca Giorgi <lgiorgi@redhat.com>

### CODE REVIEW

**Review by bdattoma — APPROVED** [review]

**Review by mwaykole — DISMISSED** [review]

File: tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot [hunk 1]
```diff
@@ -16,18 +16,22 @@ Library   OpenShiftLibrary
 *** Variables ***
 ${JUPYTERHUB_SPAWNER_HEADER_XPATH} =
```
Comment (tarukumar) [inline tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:17]: Are we using this Variable anywhere ? @lugi0

Comment (tarukumar) [reply tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot:17]: if not i think we should remove this Variable

Comment (lugi0) [reply tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot]: done

**Review by tarukumar — APPROVED** [review]

**Comment by jgarciao** [comment]
In the keyword `Spawn Notebook` at (JupyterHubSpawner), could you increase the timeout to 60s?  It's failing for some tests

```
Spawn Notebook
    [Documentation]  Start the notebook pod spawn and wait ${spawner_timeout} seconds (DEFAULT: 600s)
    [Arguments]  ${spawner_timeout}=600 seconds
    Click Button  Start server
    Wait Until Page Contains  Starting server  60s
```

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**